docs/api/models/vertexai_gemma.md

```diff
@@ -0,0 +1,85 @@
+## `class allms.models.VertexAIGemmaModel` API
```

Comment:
Since you are updating the docs, you need to also update here: https://allms.allegro.tech/

> Google Cloud Platform VertexAI models (PaLM2, Gemini, **Gemma**);

Also, I was thinking about adding to our README in master the list of supported models. I believe it's a "catchy" information that the people in the internet may be interested in when stumbling upon our repository. I will make a PR with this branch as target.

Reply:
> Since you are updating the docs, you need to also update here: https://allms.allegro.tech/
> 
> > Google Cloud Platform VertexAI models (PaLM2, Gemini, **Gemma**);

Gemma is already added there, one line below the line you quoted 😃